domains/keyruu.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Keyruu aka Lucas",
+  "repo": "https://github.com/keyruu/traversetown-htmx",
+  "owner": {
+    "username": "keyruu",
+    "email": "me@keyruu.de"
+  },
+  "record": {
+    "URL": "https://keyruu.de"
+  },
+  "proxied": true
```

This needs to be removed.
```suggestion
  }
```